pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -185,6 +185,17 @@ def tickValues(self, minVal, maxVal, minSpc):
              autoSkip=[1, 5, 10, 25])
 ], "99:99:99")
 
+
+def get_utc_offset():
```

Comment:
Can you rename to camelCase `getOffsetFromUtc` which should be more in-line w/ the Qt API: http://doc.qt.io/qt-5/qtimezone.html#offsetFromUtc

Reply:
thanks, done